tree/tree/src/RFriendInfo.cxx

```diff
@@ -18,11 +18,15 @@ namespace TreeUtils {
 /// \param[in] treeName Name of the tree.
 /// \param[in] fileNameGlob Path to the file. Refer to TChain::Add for globbing rules.
 /// \param[in] alias Alias for this friend.
-void RFriendInfo::AddFriend(const std::string &treeName, const std::string &fileNameGlob, const std::string &alias)
+/// \param[in] nEntries Number of entries for this friend.
+void RFriendInfo::AddFriend(const std::string &treeName, const std::string &fileNameGlob, const std::string &alias,
+                            std::int64_t nEntries)
 {
    fFriendNames.emplace_back(std::make_pair(treeName, alias));
    fFriendFileNames.emplace_back(std::vector<std::string>{fileNameGlob});
    fFriendChainSubNames.emplace_back();
+
```

Comment:
why the newline? it makes the reader think the next line is different/conceptually separate from the ones above but afaict it's not